src/main/java/com/mkopec/ticketbookingapp/service/TicketTypeService.java

```diff
@@ -0,0 +1,27 @@
+package com.mkopec.ticketbookingapp.service;
+
+import com.mkopec.ticketbookingapp.domain.TicketType;
+import com.mkopec.ticketbookingapp.repository.TicketTypeRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class TicketTypeService {
```

Comment:
czy to jest dobrze dobrana nazwa dla service'u, który liczy cenę rezerwacji?

Reply:
Ta nazwa powinna być inna, bardziej precyzyjna np TicketPaymentService